app.rb

```diff
@@ -1,5 +1,6 @@
 require 'sequel'
 require 'sinatra/base'
+require_relative './lib/mac_formatter.rb'
```

Comment:
Don't think it needs to be relative.

Reply:
Was going to do the require_all once we have the dependencies but yeah you are right